src/System.IO.Ports/tests/SerialPort/AbortOnError.cs

```diff
@@ -28,7 +28,7 @@ public class AbortOnError : PortsTest
         /// 3. Verify that the fAbortOnError flag is clear
         /// 
         /// </summary>
-        [ConditionalFact(nameof(HasOneSerialPort))]
+        [ConditionalFact(nameof(HasOneSerialPort), nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotUap))]
```

Comment:
Sure, it requires reflection - I want to look at it as a separate issue since simply using `InternalsVisibleTo` is not something that I saw in the corefx tests.